source/common/grpc/common.h

```diff
@@ -148,6 +154,21 @@ class Common {
    */
   static std::string typeUrl(const std::string& qualified_name);
 
+  /**
+   * BUild grpc::ByteBuffer which aliases the data in a Buffer::InstancePtr.
+   * @param bufferInstance source data container.
+   * @return byteBuffer target container aliased to the data in Buffer::Instance and owning the
+   * Buffer::Instance.
+   */
+  static grpc::ByteBuffer makeByteBuffer(Buffer::InstancePtr bufferInstance);
```

Comment:
Can you split these into a separate PR (with their tests)? They are self contained helpers that would be easier to review in isolation.

Reply:
Let's get the API settled, then I will split these and follow up on this thread.